src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
+  box-sizing: border-box;
```

Comment:
Avoid using the `box-sizing` property on specific elements like `body`. Instead, apply it globally using the `*` selector to ensure that all elements are affected. This is an exception to the rule against using the `*` selector, as `box-sizing` is commonly applied globally for consistent sizing.